Epsilon.Abstractions/Components/IComponentManager.cs

```diff
@@ -0,0 +1,6 @@
+namespace Epsilon.Abstractions.Components;
+
+public interface IComponentManager
```

Comment:
Not a big thing, but the name manager was reserved for components that require both read and write. Essentially every component is read only (which is how it was intended), but this isn't possible for pages. Therefore, I'd prefer that this is named `ComponentLoader`